docs/cli/commands.md

```diff
@@ -0,0 +1,561 @@
+---
+id: commands
+title: Commands
+description: Descriptions of all commands available in the CLI.
+---
+
+This page lists all commands provided by the Backstage CLI, what they're for,
+and where to use them.
+
+The documentation for each command begins with specifying its scope, this
+indicates where the command should be used by selecting from the following list:
+
+- `app` - A frontend app package, such as `packages/app`.
+- `backend` - A backend package, such as `packages/backend`.
+- `frontend-plugin` - A frontend plugin package.
+- `backend-plugin` - A backend plugin package.
+- `root` - The monorepo root.
+- `any` - Any kind of package, but not the repo root.
+
+## help
+
+This command displays you a help summary or detailed help screens for each
+command. Below is a cleaned up output of `yarn backstage-cli --help`.
+
+```text
+app:build                Build an app for a production release
+app:diff                 Diff an existing app with the creation template
+app:serve                Serve an app for local development
+
+backend:build            Build a backend plugin
+backend:build-image      Bundles the package into a docker image
+backend:dev              Start local development server with HMR for the backend
```

Comment:
As in `backend:serve`? Maybe, it is more about running the backend rather than serving it, but better symmetry ofc. Separate PR either way since it requires deprecation